src/streamlink_cli/main.py

```diff
@@ -981,7 +981,7 @@ def setup_plugin_options():
 def check_root():
     if hasattr(os, "getuid"):
         if os.geteuid() == 0:
-            console.logger.info("streamlink is running as root! Be careful!")
+            console.logger.info("Streamlink is running as root! Be careful!")
```

Comment:
I think this was meant to not be capitalized here, because it's referring to the `streamlink` entry script on *nix systems. This was the previous message:
https://github.com/streamlink/streamlink/commit/5f9045aed303de87a2049084a0a397877710fd90#diff-35aaec566eb720d1ce55460c85bf4ce9L841